cli/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "strictNullChecks": true,
+    "strict": true,
```

`strictNullChecks` is already part of the `strict` "family"
```suggestion
    "strict": true,
```